### ISSUE TITLE
Missing 'file' folder in %APPDATA%\Code\User\globalStorage\rangav.vscode-thunder-client

### ISSUE THREAD

**Issue by thabing** [rangav/thunder-client-support#880]
**Describe the bug**

My first use of the client after installing it was testing an API that returns binary PDF files, it seemingly worked, but the Thunderbird Client was responding with an error:

`Error when sending request: ENOENT: no such file or directory, open 'c:\Users\xxxx\AppData\Roaming\Code\User\globalStorage\rangav.vscode-thunder-client\files\file_6da83e66-40be-4aee-80d9-fbc228df61de.pdf'`

**To Reproduce**

**Expected behavior**
Some way to open or save the binary file.

**Platform:** 
 - OS: Windows 10
 - vscode version: 1.73.1
 - extension version: 1.20.1

**Solution:**
On a hunch, I looked in the 'c:\Users\xxxx\AppData\Roaming\Code\User\globalStorage\rangav.vscode-thunder-client\' folder, but the 'files' folder was missing.  After creating this subfolder, the request worked as expected, and the client provide the option to save or open the file:

```
The response is binary file, display not supported
Save File
Open in Code
```
Maybe this is related to #635 ??

**Comment by rangav** [comment]
@thabing thanks for reporting, can you re-start `vscode` and try again.

**Comment by thabing** [comment]
@rangav thanks.  After restarting vscode, it seems to have automatically created the '...\rangav.vscode-thunder-client\files' folder and works as expected.

**Comment by rangav** [comment]
Thanks for confirmation, Actually I found the bug related to this, it's fixed now.

**Comment by rangav** [comment]
This Bug is fixed now and published to marketplace, please update to `v2.0.0`

See all features released
https://github.com/rangav/thunder-client-support/releases/tag/v2.0.0

Please let me know your feedback.
